source/Grid/defaultCellRangeRenderer.js

```diff
@@ -40,7 +97,8 @@ export default function defaultCellRangeRenderer ({
         isScrolling,
         key,
         rowIndex,
-        style
+        style,
+        visibility: findOverlap(gridDimensions, style)
```

Comment:
Thinking about this more, we could even deprecated (but not remove, for backwards compatibility) `isScrolling` in favor of `visibility` and set the state to `scrolling` during scrolling, so like:

```js
visibility: isScrolling ? 'scrolling' : findOverlap(gridDimensions, style)
```

This also removes the ambiguity of the values that would otherwise need to be set for `x` and `y` during scrolling.